### PR TITLE
vendor: mysql x-protocol protobuf files

### PR DESCRIPTION
**Pull request by jackysp**
Add mysql x-protocol protobuf files.
@hicqu @tiancaiamao PTAL

### CODE REVIEW

File: glide.lock [hunk 1]
```diff
@@ -193,7 +203,7 @@ imports:
   subpackages:
   - rate
 - name: google.golang.org/genproto
-  version: 6b7d9516179cd47f4714cfeb0103ad1dede756c4
+  version: ee236bd376b077c7a89f260c026c4735b195e459
```
Comment (jackysp) [inline glide.lock]: Why this changed?

File: glide.yaml [hunk 2]
```diff
@@ -75,6 +75,10 @@ import:
   - http2/hpack
   - internal/timeseries
   - trace
+- package: google.golang.org/genproto
```
Comment (jackysp) [inline glide.yaml]: Lock genproto version

**Comment by hanfei1991** [comment]
LGTM

**Comment by hicqu** [comment]
LGTM

**Pull request merged**